packages/e2e-tests/specs/block-transforms.test.js

```diff
@@ -96,91 +97,101 @@ const getTransformResult = async ( blockContent, transformName ) => {
-		)
-	);
-
 	const transformStructure = {};
-	beforeAll( async () => {
-		await createNewPost();
-
-		for ( const fileBase of fileBasenames ) {
-			const structure = await getTransformStructureFromFile(
-				fileBase
-			);
-			if ( ! structure ) {
-				continue;
-			}
-			transformStructure[ fileBase ] = structure;
-			await setPostContent( '' );
-			await page.click( '.editor-post-title .editor-post-title__block' );
-		}
-	} );
-
-	it( 'should contain the expected transforms', async () => {
-		const transforms = mapValues(
-			pickBy(
-				transformStructure,
-				( { availableTransforms } ) => availableTransforms,
-			),
-			( { availableTransforms, originalBlock } ) => {
-				return { originalBlock, availableTransforms };
-			}
+	if ( ! process.env.POPULAR_PLUGINS ) {
+		// Todo: Remove the filter as soon as all fixtures are corrected,
+		// and its direct usage on the editor does not trigger errors.
+		// Currently some fixtures trigger errors (mainly media related)
+		// because when loaded in the editor,
+		// some requests are triggered that have a 404 response.
+		const fileBasenames = filter(
+			getAvailableBlockFixturesBasenames(),
+			( basename ) => (
+				! some(
+					[
+						'core__image',
+						'core__gallery',
+						'core__video',
+						'core__file',
+						'core__media-text',
+						'core__audio',
+						'core__cover',
+					],
+					( exclude ) => basename.startsWith( exclude )
+				)
+			)
 		);
-		expect(
-			transforms
-		).toEqual( EXPECTED_TRANSFORMS );
-	} );
 
-	describe( 'correctly transform', () => {
 		beforeAll( async () => {
 			await createNewPost();
+
+			for ( const fileBase of fileBasenames ) {
+				const structure = await getTransformStructureFromFile(
+					fileBase
+				);
+				if ( ! structure ) {
+					continue;
+				}
+				transformStructure[ fileBase ] = structure;
+				await setPostContent( '' );
+				await page.click( '.editor-post-title .editor-post-title__block' );
+			}
 		} );
 
-		beforeEach( async () => {
-			await setPostContent( '' );
-			await page.click( '.editor-post-title .editor-post-title__block' );
+		it( 'should contain the expected transforms', async () => {
+			const transforms = mapValues(
+				pickBy(
+					transformStructure,
+					( { availableTransforms } ) => availableTransforms,
+				),
+				( { availableTransforms, originalBlock } ) => {
+					return { originalBlock, availableTransforms };
+				}
+			);
+			expect(
+				transforms
+			).toEqual( EXPECTED_TRANSFORMS );
 		} );
+	}
 
+	describe( 'correctly transform', () => {
 		const testTable = flatMap(
 			EXPECTED_TRANSFORMS,
 			( { originalBlock, availableTransforms }, fixture ) => (
 				map(
 					availableTransforms,
-					( distinationBlock ) => ( [
+					( destinationBlock ) => ( [
 						originalBlock,
 						fixture,
-						distinationBlock,
+						destinationBlock,
 					] )
 				)
 			)
 		);
 
+		const testMessageFormat = 'block %s in fixture %s into the %s block';
+
+		if ( process.env.POPULAR_PLUGINS ) {
```

Comment:
This test is hard to follow at the moment. 

Is it possible to divide this test suite into 2 groups? One that runs always and the other one which runs only when `POPULAR_PLUGINS` is disabled?

Reply:
Thank you for the suggestion the code was updated and simplified to follow it :)